backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -49,11 +50,37 @@
 from tracim_backend.models.data import ActionDescription
 from tracim_backend.models.data import UserRoleInWorkspace
 
-
 FIELD_LANG_DESC = "User langage in ISO 639 format. See https://fr.wikipedia.org/wiki/ISO_639"
 FIELD_PROFILE_DESC = "Profile of the user. The profile is Tracim wide."
 FIELD_TIMEZONE_DESC = "Timezone as in tz database format"
 
+
+class SimpleFileSchema(marshmallow.Schema):
+    """
+    Just a simple schema for file
+    """
+    # TODO - G.M - 2018-10-09 - Set required to True, actually disable because
+    # activating it make it failed due to "is not iterable issue.
+    files = marshmallow.fields.Raw(required=False, description='a file')
+
+    @post_load
+    def create_file(self, data):
+        return SimpleFile(**data)
+
+
+class FileCreationFormSchema(marshmallow.Schema):
+    parent_id = marshmallow.fields.Int(
+        example=2,
+        default=0,
+        validate=Range(min=0, error="Value must be positive or 0"),
+        allow_none=True
+    )
+
+    @post_load
+    def create_set_email_object(self, data):
+        return FileCreation(**data)
```

Comment:
I really do not understand the naming strategy and the associated logic: FieCreation is a container with a unique property parent_id, and it is set through the call to a `create_set_email_object()` method

Reply:
@inkhey Some information about this remark ?

Reply:
@buxx create_set_email_object was renamed in order to make  sense.

Reply:
okay